testData/solutions/win32.json5

```diff
@@ -1691,396 +1773,434 @@
+                            "title": "Scheme",
+                            "description": "Whether or not to intercept CreateDIBSection Note that this value is only read once per JAWS session at startup thus changing this value while JAWS is running has no effect.",
+                            "type": "string",
                             "enum": [
-                                0,
-                                1
+                                "Attributes and Colors",
+                                "Classic",
+                                "Classic (Attributes and Font info)",
+                                "Classic (Attributes)",
+                                "Classic (Attributes, Font and Color)",
+                                "Classic With Quotes",
+                                "Colors",
+                                "Indent (tab is 0.125 inches Script Manager)",
+                                "Indent (Tab is Eight Spaces)",
+                                "Indent (Tab is Four Spaces)",
+                                "Indent (Tab is Two Spaces)",
+                                "Kindle",
+                                "ProofReading (Attributes and Font Info)",
+                                "ProofReading (Attributes)",
+                                "ProofReading (Attributes, Font Info and color)",
+                                "SayAll Text Only",
+                                "SayAll Text With Sounds",
+                                "Wsual Studio .Net",
+                                "Visual Studio .Net (Colors)",
+                                "Wsual Studio .Net (Indent)",
+                                "Web RentACrowd",
+                                "Word Classic"
+                            ],
+                            "enumLabels": [
+                                "Attributes and Colors",
+                                "Classic",
+                                "Classic (Attributes and Font info)",
+                                "Classic (Attributes)",
+                                "Classic (Attributes, Font and Color)",
+                                "Classic With Quotes",
+                                "Colors",
+                                "Indent (tab is 0.125 inches Script Manager)",
+                                "Indent (Tab is Eight Spaces)",
+                                "Indent (Tab is Four Spaces)",
+                                "Indent (Tab is Two Spaces)",
+                                "Kindle",
+                                "ProofReading (Attributes and Font Info)",
+                                "ProofReading (Attributes)",
+                                "ProofReading (Attributes, Font Info and color)",
+                                "SayAll Text Only",
+                                "SayAll Text With Sounds",
+                                "Wsual Studio .Net",
```

Comment:
Funny.  Search for "Wsual" and replace with "Visual" throughout.

Reply:
Hahahaha. Searched and replaced.